ironfish-cli/src/commands/wallet/notes/combine.ts

```diff
@@ -469,6 +469,18 @@ export class CombineNotesCommand extends IronfishCommand {
       raw = RawTransactionSerde.deserialize(bytes)
     }
 
+    // This allows for a single note output.
+    const amount = amountIncludingFees - raw.fee
+    params.outputs[0].amount = CurrencyUtils.encode(amount)
+    params.fee = raw.fee ? CurrencyUtils.encode(raw.fee) : null
```

Comment:
nit - do we need the ternary operator here? I think `fee` is always non-null on a `RawTransaction`

Reply:
Removed! Thank you!